redhat-cop/uncontained.io
# [WIP] Add article: Monitoring Microservices With The Prometheus Operator

#### What is this PR About?

How to monitor applications with the Prometheus Operator

This is a common question and task in the field, due to the fact that the CMO does not support any changes being made to it, deploying and managing the CoreOS Prometheus Operator is the alternative.

This is a WIP and am requesting feedback on its current state, on content and formatting

#### How should we test or review this PR?

Verify the content, build/preview.

Testing the instructions on OCP is welcomed, I'd like to keep the barrier of entry low.

#### Is there a relevant Trello card or Github issue open for this?

No, I've accumulated a good amount of material in the field on this, enough that it was worth submitting an article outright I thought.

#### Who would you like to review this?

cc: @redhat-cop/cant-contain-this

'''''

* [✅] Have you followed the
[contributing
guidelines?](https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.adoc)
* [✅] Have you explained what your changes do, and why they add value to
the uncontained.io guides?

'''''

*Please note: we may close your PR without comment if you do not check
the boxes above and provide ALL requested information.*
